duckduckgo/Android
# Update onboarding assets with new copy

Task/Issue URL: https://app.asana.com/0/72649045549333/631150607895653
Tech Design URL: 
CC: 

**Description**:
Updated with new assets from design

**Steps to test this PR**:
1. Visit 3rd screen in onboarding and check it looks ok
 (needs Nougat or newer; variant depending, 25% chance of seeing it. clear data and try again if it's not visible)


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
